.github/workflows/master.yml

```diff
@@ -0,0 +1,63 @@
+name: Master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  bump-version:
+    name: Bump Version
+    if: github.event.commits[0].author.name != 'flyte-bot'
```

Comment:
Can we use head_commit from the event instead.
https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push

Reply:
done